lib/order.rb

```diff
@@ -0,0 +1,68 @@
+require "csv"
+require_relative "customer"
+
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    if (@fulfillment_status != :pending && @fulfillment_status != :shipped && @fulfillment_status != :paid && @fulfillment_status != :processing && @fulfillment_status != :complete)
```

Comment:
There is probably a shorter way to write this conditional that I would encourage because this line is very long!